src/Symfony/Component/Profiler/ProfileData/ProfileDataInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Profiler\ProfileData;
+
+/**
+ * ProfileDataInterface.
+ *
+ * @author Jelte Steijaert <jelte@khepri.be>
+ */
+interface ProfileDataInterface
```

Comment:
I think this interface should extend `\Serializable`. Any profile data should be serializable.